song/services/likeService.js

```diff
@@ -0,0 +1,10 @@
+const likeDao = require("../models/likeDao");
+
+const like = async (userId, postId) => {
+  const like = await likeDao.like(userId, postId);
+  return like;
```

Comment:
```suggestion
  return likeDao.like(userId, postId);
```
굳이 변수 선언 및 동기처리를 통해 두 줄로 작성될 필요가 없는 코드입니다.